src/alire/alire-releases.adb

```diff
@@ -884,8 +887,10 @@ package body Alire.Releases is
         "short_description: " & Utils.YAML.YAML_Stringify (R.Description) &
         ASCII.LF &
         "dependencies: " & R.Dependencies.To_YAML & ASCII.LF &
-        "config_variables: " & Props_To_YAML (R.Config_Variables) & ASCII.LF &
-        "config_settings: " & Props_To_YAML (R.Config_Settings) & ASCII.LF;
+        "configuration.variables: " &
```

Comment:
I am not sure this will work for YAML. But I can change it later if needed.

Reply:
Ah, because those are YAML dictionary keys... Got it.